app/src/main/kotlin/com/simplemobiletools/calculator/helpers/MoneyCalculator.kt

```diff
@@ -0,0 +1,6 @@
+package com.simplemobiletools.calculator.helpers
+
+interface MoneyCalculator {
+    fun spawnTaxModal()
+    fun spawnCurrencyModal()
```

Comment:
Why do you need an interface to implement this method considering you don't need the geo-location callback?

Reply:
Same thought here. You can literally just go from view to controller no need to go the other way around. If I understand correctly what you are doing is going from view-->controller-->view to spawn a dialog.

Reply:
The MoneyCalculatorImpl has a moneyCallback using this interface to the MoneyActivity where the methods are implemented. This removes the MoneyCalculatorImpl's dependency on a MoneyActivity.

Reply:
But why is a callback necessary? Why not just have the button trigger the dialog directly? Is something getting checked beforehand?

Reply:
It may not be necessary. @todd829 Can you consider this point when you do your refactoring?